app/jobs/send_accepted_rejected_notification_job.rb

```diff
@@ -0,0 +1,32 @@
+class SendAcceptedRejectedNotificationJob < ApplicationJob
+  def perform(submission_ids)
+    submission_ids.each do |id|
+      send_notification(id)
+    end
+  end
+
+  def send_notification(efile_submission_id)
+    submission = EfileSubmission.for_state_filing.find(efile_submission_id)
+    if submission.nil?
+      puts "*****Error: No efile submission found with id #{efile_submission_id}"
```

Comment:
Should we use the rails logger here rather than puts ?

Reply:
@tofarr ooo yes something like `Rails.logger.info "*****Error: No efile submission found with id #{efile_submission_id}"`?